src/rgw/rgw_kafka.cc

```diff
@@ -588,13 +596,13 @@ class Manager {
       return false;
     }
 
-    auto conn = std::make_unique<connection_t>(cct, broker, use_ssl, verify_ssl, ca_location, user, password, mechanism);
+    auto conn = std::make_unique<connection_t>(cct, broker_list, use_ssl, verify_ssl, ca_location, user, password, mechanism);
```

Comment:
need check this line

Reply:
this is ok, since we use this value when we pas the list of brokers to librdkafka:
https://github.com/ceph/ceph/blob/b08530e8b2374c650b84d88e3293042c2ffcc693/src/rgw/rgw_kafka.cc#L248